dpgen2/op/prep_dp_optim.py

```diff
@@ -0,0 +1,168 @@
+import json
+import logging
+import pickle
+import shutil
+from pathlib import (
+    Path,
+)
+from typing import (
+    List,
+    Tuple,
+)
+
+from dflow.python import (
+    OP,
+    OPIO,
+    Artifact,
+    BigParameter,
+    OPIOSign,
+    Parameter,
+    TransientError,
+)
+
+from dpgen2.constants import (
+    calypso_check_opt_file,
+    calypso_opt_dir_name,
+    calypso_run_opt_file,
+    model_name_pattern,
+)
+from dpgen2.exploration.task import (
+    ExplorationTaskGroup,
+)
+from dpgen2.utils import (
+    BinaryFileInput,
+    set_directory,
+)
+from dpgen2.utils.run_command import (
+    run_command,
+)
+
+
+class PrepDPOptim(OP):
```

Comment:
It is not a general optimization op with DP. I would suggest renaming it to e.g. `PrepDPCalyOptim`